content/benefits/benefits-documents/release-notes/2024-05-22.md

```diff
@@ -0,0 +1,11 @@
+This release includes 2 new changes to the Benefits Documents API: 
+
+In order to enable dependents of deceased Veterans to upload and search for benefits related documents, we have added a new participantId parameter to the request objects for the following endpoints:
+* /documents
+* /documents/search 
+
+The participantId value is used to identify veterans and/or their dependents, and will be used instead of a file number. 
```

Comment:
Is it the change from from file number to participant id that enables the functionality? (uploading/searching that includes dpendents)

Reply:
Short answer - Yes
Longer answer - 
The participant ID value allows us to identify a dependent, as they will have their own value for this, and (after validating their relationship to the veteran) grant them access to the contents of the eFolder. 

The file number value does not allow us to make this distinction between veteran and dependent. Dependents do not have this value, thus they can't use the EVSS Documents Service, where this problem was first identified. Firefly was asked to build a solution for this, and in turn facilitate VA teams' migration to our service, which they're doing anyway.

Reply:
the proposed copy says that participantId will be used `instead` of fileNumber. Looking at the schema on the portal, I see both fileNumber and participantId. Can both still be used?

Reply:
The service accepts either value, and will default to the participant ID if both are provided. In order for a dependent to have access, however, they will need to submit their participant ID.